app/models/ArrivalStatus.scala

```diff
@@ -67,6 +69,13 @@ object ArrivalStatus extends Enumerable.Implicits with MongoDateTimeFormats {
     }
   }
 
+  case object XMLSubmissionNegativeAcknowledgement extends ArrivalStatus {
+    override def transition(messageReceived: MessageReceivedEvent): Either[TransitionError, ArrivalStatus] = messageReceived match {
+      case MessageReceivedEvent.XMLSubmissionNegativeAcknowledgement => Right(XMLSubmissionNegativeAcknowledgement)
+      case _                                                         => Left(TransitionError(s"Tried to transition from XMLSubmissionNegativeAcknowledgement to $messageReceived."))
```

Comment:
Are we not allowing any more messages for a movement after a 917?

Reply:
As of now it is 917 response based on the notifications we have received for live issues

Reply:
This does raise alarm bells - do we have any tickets or intention to follow this up? We've effectively ended an arrival permanently with this transition and forced a new one, it might even be worse because if you try to resubmit with the same MRN, it will try to merge, transition and kill that arrival too.

Reply:
Also, if this is going to be included, then we should confirm this behaviour in an acceptance test.

Reply:
This is parity with the current system, if there is an IE917 then that's the end of the movement and then the submitting system needs to raise a new movement. It would be nice to group it together on our end for APIs / Frontends to consume but that hasn't been done as yet. 

This is also in effect the same as the current process for an IE008 with the FE where for some error types we don't group together under the same movement. 

This isn't well known and understood this branch of errors and what the followup should be so this is open to change as we learn more about the process after receipt of an IE917